public/app/features/search/components/DashboardList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { useAsync } from 'react-use';
+import { connect, MapStateToProps } from 'react-redux';
+import { NavModel } from '@grafana/data';
+import { getLocationSrv } from '@grafana/runtime';
+import { StoreState } from 'app/types';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { getRouteParams } from 'app/core/selectors/location';
+import Page from 'app/core/components/Page/Page';
+import locationUtil from 'app/core/utils/location_util';
+import { backendSrv } from 'app/core/services/backend_srv';
+import { ManageDashboards } from './ManageDashboards';
+
+interface Props {
+  navModel: NavModel;
+  uid?: string;
+}
+
+export const DashboardList: FC<Props> = ({ navModel, uid }) => {
```

Comment:
We usually name this top level container / page component with a Page suffix, so DashboardListPage (same for the filename)

Reply:
oh and use React.memo here so the whole page does not re-render on every redux state change